scripts/ripe-atlas

```diff
@@ -68,12 +68,18 @@ class RipeAtlas(object):
 
         try:
 
-            cmd = __import__(
+            module = __import__(
                 "ripe.atlas.tools.commands." + self.command,
                 globals(),
                 locals(),
                 [self.command]
-            ).Command(*self.args, **self.kwargs)
+            )
+
+            if hasattr(module, "Factory"):
+                cmd = module.Factory(*self.args, **self.kwargs).build()
+            else:
+                cmd = module.Command(*self.args, **self.kwargs)
+
             cmd.init_args()
```

Comment:
Everything seems fine, structure-wise. Only two small things, 1) let's add some comment on this code, so people who look at it know the reasoning for Factoring, 2) maybe build should be create? i think it describes better what it does and I think is what usually people use for factory classes,no?
I will try to look later today the other errors.